containers/hydra/messager.py

```diff
@@ -18,90 +19,108 @@
 
 # quick old-school way to check needed parameters
-if (SLACKMESSAGE=='' or  SLACKCONFIGURATIONFILE=='' ):
-        print("\n---> MISSING ARGUMENTS!!\n ")
-        parser.print_help()
-        sys.exit(2)
+if (SLACKMESSAGE == '' or SLACKCONFIGURATIONFILE == ''):
+    print("\n---> MISSING ARGUMENTS!!\n ")
+    parser.print_help()
+    sys.exit(2)
 
 file_exists = os.path.exists(SLACKCONFIGURATIONFILE)
 
-if (file_exists):
-
-        SLACKMESSAGE="" # forming first message part here, not using passed argument
-
-        print ("Slack configuration file exists. Going to do the messaging",file=sys.stderr)
-        file = open(SLACKCONFIGURATIONFILE, "r")
-        config_array=file.read().split()
-        slackchannel=str(config_array[1])
-        slacktoken=str(config_array[0])
-
-        hydraserver = os.getenv("POSTBUILD_SERVER")
-        if hydraserver == None:
-            print ("No Hydra server defined",file=sys.stderr)
-            HYDRASERVER=""
-        else:
-            HYDRASERVER="\nHydra server:"+hydraserver
-
-        hydradata= os.getenv("HYDRA_JSON")
-        if hydradata == None:
-            print ("No Hydra JSON defined",file=sys.stderr)
-        else:
-            with open(hydradata) as jsonf:
-                binfo = json.load(jsonf)
-                #prettyinfo=json.dumps(binfo,indent=3)
-                buildjob=str(binfo['job'])
-                buildstatus=str(binfo['buildStatus'])
-                buildnumber=str(binfo['build'])
-                buildproject=str(binfo['project'])
-
-                if (int(buildstatus)==0):
-                    SLACKMESSAGE="OK Build !!!"
-
-                else:
-                    SLACKMESSAGE="Broken Build !!!"
-
-                SLACKMESSAGE=SLACKMESSAGE+HYDRASERVER+"\nHydra build:"+str(buildjob)+"\nStatus:"+buildstatus+"\nNumber:"+buildnumber+"\nProject:"+buildproject
-
-        try:
-            client = slack.WebClient(token=slacktoken)
-            client.chat_postMessage(channel=slackchannel, text=SLACKMESSAGE)
-
-        except Exception as e:
-            print(("Slacking failed! Check your channel name?, error: %s" % e))
-            sys.exit(1)
+if file_exists:
+
+    SLACKMESSAGE = ""  # forming first message part here, not using passed argument
+
+    print("Slack configuration file exists. Going to do the messaging", file=sys.stderr)
+    file = open(SLACKCONFIGURATIONFILE, "r", encoding="utf-8")
+    config_array = file.read().split()
+    slackchannel = str(config_array[1])
+    slacktoken = str(config_array[0])
+
+    hydraserver = os.getenv("POSTBUILD_SERVER")
+    if hydraserver is None:
+        print("No Hydra server defined", file=sys.stderr)
+        HYDRASERVER = ""
+    else:
+        HYDRASERVER = "\nHydra server:"+hydraserver
+
+    hydradata = os.getenv("HYDRA_JSON")
+    if hydradata is None:
+        print("No Hydra JSON defined", file=sys.stderr)
+    else:
+        with open(hydradata) as jsonf:
+            binfo = json.load(jsonf)
```

Comment:
We can consider using more descriptive variable names,  For example, instead of binfo, you might can use build_info

Reply:
Will look this when next change issue under constructions